packages/ui-app/src/Params/types.ts

```diff
@@ -15,6 +15,9 @@ export type RawParam$Values = RawParam$Value | RawParam$ValueArray;
 
 export type RawParam = {
   isValid: boolean,
+  info?: number,
```

Comment:
Ok, if we have to do this, I would expect it to actually have something like a `typeDef` on this object, or even `type` containing the actual type information.

Reply:
oh i see what you mean now, i found a similar example here polkadot-js/api/types/codec/createType.d.ts